### PR TITLE
Update Ui.png, userguide update

### PR DESCRIPTION
**Pull request by zhihong8888**
- Updated all the command field names beside the prefix to match the command format.
- Updated some parts of user guide.
- Updated command usage for menu bar to look better

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -349,18 +348,18 @@ Format: `clearExpenses` or `ce`
 
 Add a new schedule for a employee.
 
-Format: `addSchedule id/EMPLOYEE_ID d/DATE t/TYPE` or `aw id/EMPLOYEE_ID d/DATE t/TYPE`
+Format: `addSchedule id/EMPLOYEEID d/DATE t/TYPE` or `aw id/EMPLOYEEID d/DATE t/TYPE`
```
Comment (LimYiSheng) [inline docs/UserGuide.adoc]: For the alias method, shouldn't it be "as" instead of "aw"?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -630,13 +637,24 @@ The `undo` command fails as there are no undoable commands executed previously
...
 Undoable commands: those commands that modify CHRS content.
 For commands similar functions to +
 (list*, find*, filter*, select*, calculate*)
 the command will not be able to undo or redo).
 ====
+=== Redoing the previously undone command : `redo`
+
+Reverses the most recent `undo` command.
+
+Format: `redo`
```
Comment (LimYiSheng) [inline docs/UserGuide.adoc]: Redo command has been covered in the user guide already, this seems to be a repeated coverage. Do check and recommit

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -670,6 +688,14 @@ the command will not be able to undo or redo.
...
 // end::undoredo[]
 
+[NOTE]
+====
+Redoable commands: those commands that modify CHRS content.
+For commands similar functions to +
+(list*, find*, filter*, select*, calculate*)
+the command will not be able to undo or redo.
+====
+
```
Comment (LimYiSheng) [inline docs/UserGuide.adoc]: This note is also a repeated note within the userguide, there already exists one (exact same) for redo

File: docs/UserGuide.adoc (hunk 2 above)
Comment (zhihong8888) [reply docs/UserGuide.adoc]: fixed, yup i dunno y it duplicated twice

**Pull request merged**